src/ApiCall.ts

```diff
@@ -269,14 +274,80 @@ export async function getAccessToken() {
   }
 }
 
+export async function getUserAccessToken() {
+  const config: any = await getConfig()
+  consoleLog(config, 'API call - getUserAccessToken');
+  var accessToken: string | null = null;
+  var appId = JSON.parse(await EncryptedStorage.getItem(appIdC) as string);
+  if (config.environment == 'development') {
+    accessToken = await EncryptedStorage.getItem(appId + user_accessToken_development);
+  } else if (config.environment == 'staging') {
+    accessToken = await EncryptedStorage.getItem(appId + user_accessToken_staging);
+  } else {
+    accessToken = await EncryptedStorage.getItem(appId + user_accessToken_prod);
+  }
+  const auth = JSON.parse(await EncryptedStorage.getItem(appId + user_auth) as string);
+  if (config != 'undefined' && config != null) {
+    if (typeof accessToken !== 'undefined' && accessToken != null) {
+      const accessToken_Obj = JSON.parse(accessToken);
+      const createdTime = accessToken_Obj.created_at + 6000;
```

Comment:
Did you see my comment in Slack yesterday? There's no created_at returned when we first set refresh token. Is this something we can set as the current time when we get refresh token maybe? If I'm reading this correctly it will always refresh token every time?